### PR TITLE
add keys relative to home folder

### LINKED ISSUE #109: eigenlayer operator keys create to use directory relative to $HOME
**Is your feature request related to a problem? Please describe.**
Currently `eigenlayer operator keys create` creates keys in the folder relative to the current working directory. The list command will fail if we use list command in some other directory

**Describe the solution you'd like**
- Use a folder relative to home directory (maybe which the install command is using)

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by shrimalmadhur**
Fixes | Closes | Resolves #

#109

## Changes:
- Create/Import/List keys relative to home folder

## Types of changes

_Leave on the following list the types of changes introduced by this PR and remove
the ones that don't apply. Please also remove this line._

- New feature (non-breaking change which adds functionality)

## Testing

**Requires testing** Yes/No

**In case you checked yes, did you write tests?** Yes/No

**Comments about testing , should you have some** (optional)

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by AntiD2ta — APPROVED** [review]
Approved!

**Review by AntiD2ta — DISMISSED** [review]
Please update the changelog. The rest LGTM

File: cli/operator/keys/import_test.go [hunk 1]
```diff
@@ -86,7 +92,7 @@ func TestImportCmd(t *testing.T) {
 			promptMock: func(p *prompterMock.MockPrompter) {
 				p.EXPECT().InputHiddenString(gomock.Any(), gomock.Any(), gomock.Any()).Return("", nil)
 			},
-			keyPath: OperatorKeyFolder + "/test.ecdsa.key.json",
+			keyPath: filepath.Clean(homePath + "/" + OperatorKeyFolder + "/test.ecdsa.key.json"),
```
Comment (samlaf) [inline cli/operator/keys/import_test.go]: use filepath.join instead of `+ "/"`

File: cli/operator/keys/import_test.go [hunk 2]
```diff
@@ -95,7 +101,7 @@ func TestImportCmd(t *testing.T) {
...
```
Comment (samlaf) [inline cli/operator/keys/import_test.go]: use filepath.join instead of `+ "/"`

File: cli/operator/keys/import_test.go [hunk 3]
```diff
@@ -104,13 +110,13 @@ func TestImportCmd(t *testing.T) {
 			promptMock: func(p *prompterMock.MockPrompter) {
 				p.EXPECT().InputHiddenString(gomock.Any(), gomock.Any(), gomock.Any()).Return("", nil)
 			},
-			keyPath: OperatorKeyFolder + "/test.bls.key.json",
+			keyPath: filepath.Clean(homePath + "/" + OperatorKeyFolder + "/test.bls.key.json"),
```
Comment (samlaf) [inline cli/operator/keys/import_test.go]: use filepath.join instead of `+ "/"`

File: cli/operator/keys/create.go [hunk 4]
```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	OperatorKeyFolder = "operator_keys"
+	OperatorKeyFolder = ".eigenlayer/operator_keys"
```
Comment (samlaf) [inline cli/operator/keys/create.go]: Rename to `OperatorKeyFolderRelToHome` or something?

Comment (shrimalmadhur) [reply cli/operator/keys/create.go]: Hmm. I don't know. this name feels a bit weird. maybe `OperatorKeySubFolder` or `OperatorKeystoreSubFolder`

Comment (shrimalmadhur) [reply cli/operator/keys/create.go]: `OperatorKeystoreSubFolder` it is

**Review by samlaf — APPROVED** [review]
LGTM

**Pull request merged**